### DISCUSSION TITLE
Linking to GitHub Discussions from the README?

### DISCUSSION THREAD

**Discussion by tvercaut** [Project-MONAI/MONAI#1360] [General]
Hi,
Should the [community](https://github.com/Project-MONAI/MONAI#community) section of the [README](https://github.com/Project-MONAI/MONAI/blob/master/README.md) already point to this Discussion tab? Or is it better to wait and see a bit first?
Tom

**Comment by tvercaut** [comment]
Answering to myself, I saw that @wyli already opened a related issue for it: https://github.com/Project-MONAI/MONAI/issues/1358

**Comment by rijobro** [comment]
Addressed with https://github.com/Project-MONAI/MONAI/pull/1364!
